nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/bruker/BrukerData.java

```diff
@@ -735,15 +735,20 @@ public String getText() {
         return text;
     }
 
+    @Override
+    public boolean arePhasesSet(int dim) {
+        Double ph0 = getParDouble("PHC0," + (dim + 1));
+        Double ph1 = getParDouble("PHC1," + (dim + 1));
+        return (ph0 != null && Math.abs(ph0) > 1.0e-9) || (ph1 != null && Math.abs(ph1) > 1.0e-9);
+    }
+
     @Override
     public double getPH0(int iDim) {
         double ph0 = 0.0;
         Double dpar;
         if ((dpar = getParDouble("PHC0," + (iDim + 1))) != null) {
             ph0 = -dpar;
-            if (iDim == 0) {
-                ph0 += 90.0;
-            } else if (iDim == 1) {
+            if (iDim == 1) {
                 ph0 += deltaPh0_2;
             }
         }
```

Comment:
after testing it seems like adding 90 to ph0 does not load correctly so have removed it. Only have tested with 1D experiments, the phasing should be tested with other experiments